src/scss/styles.scss

```diff
@@ -24,6 +24,7 @@ $focusOutline: 0px 0 0 1px $blue,
 @import 'generic/forms';
 @import 'generic/typography';
 
+@import 'components/auth0';
```

Comment:
I'm not sure if we can actually style the duo screen, from my reading of my network logs after entering my LDAP creds, auth0 redirects me to `https://auth.mozilla.auth0.com/mf` which I don't think we have any control over

Reply:
It can't affect the duo screen, but it can control the sizing of the duo iframe, which seems to be all that is necessary to have it flow correctly.

![](https://files.slack.com/files-pri/T027LFU12-F01333001UL/screen_shot_2020-05-01_at_10.55.35_am.png)